plugins/jsonld-contexts/package.json

```diff
@@ -0,0 +1 @@
+{}
```

Comment:
What is this here for?

Reply:
Gatsby plugins require a package.json file. For local plugins they recommend just having an empty object in it. It's kinda weird.

Reply:
What? That's super weird.